src/io/flutter/vmService/DartVmServiceListener.java

```diff
@@ -40,6 +40,7 @@ public void connectionOpened() {
 
   }
 
+  @SuppressWarnings("DuplicateBranchesInSwitch")
```

Comment:
Why is this needed? I don't see any dups, and I don't have a warning about it. I do have a warning about missing branches (Logging, VMFlagUpdate).

Reply:
Interesting. I don't see the missing branches warnings. The duplicated case statements refer to the cases that all just end in `break;`.